tests/expectations/alias.compat.c

```diff
@@ -0,0 +1,57 @@
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+enum Status
+#ifdef __cplusplus
+  : uint32_t
+#endif // __cplusplus
+ {
+  Ok,
+  Err,
+};
+#ifndef __cplusplus
+
```

Comment:
Spacing here in the generated code is a bit weird, may be worth fixing it if it doesn't take much effort, but maybe worth doing it in a separate PR.